docs/api-reference/extensions/data-filter-extension.md

```diff
@@ -165,6 +166,22 @@ When an object is "faded", manipulate its opacity so that it appears more transl
 Enable/disable the data filter. If the data filter is disabled, all objects are rendered.
 
 
+## Remarks
+
+### Filter precision
+
+By default, both the filter values and the filter range are uploaded to the GPU as 32-bit floats. When using very large filter values, most commonly Epoch timestamps, 32-bit float representation could lead to an error margin of >1 minute. Enabling 64-bit precision by setting `fp64: true` would allow the filter range to be evaluated more accurately. However, 64-bit support requires one extra attribute slot, edging closer to the WebGL limit of 16 attributes. Depending on the layer that the `DataFilterExtension` is used with, it may interfere with the layer's ability to use other extensions.
```

Comment:
16 attributes isn't a universal limit. It depends on the hardware: https://webglstats.com/webgl/parameter/MAX_VERTEX_ATTRIBS

I'd just say something like "requires one extra attribute slot, which increases the risk of exceeding the hardware limit on vertex attributes."